.github/workflows/build-for-release.yml

```diff
@@ -9,37 +9,51 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-latest, windows-latest, macos-latest]
+        build_name:
+          [linux-x64-musl, linux-arm64-musl, macos-arm64, windows-x64-msvc]
         include:
-          - os: ubuntu-latest
+          - build_name: linux-x64-musl
+            os: ubuntu-latest
             exe_suffix: ""
             name_suffix: ""
             target: x86_64-unknown-linux-musl
-          - os: macos-latest
+          - build_name: linux-arm64-musl
+            os: ubuntu-latest
+            exe_suffix: ""
+            name_suffix: ""
+            target: aarch64-unknown-linux-musl
+          - build_name: macos-arm64
+            os: macos-latest
             exe_suffix: ""
             name_suffix: ".arm"
             target: aarch64-apple-darwin
-          - os: windows-latest
+          - build_name: windows-x64-msvc
+            os: windows-latest
             exe_suffix: ".exe"
             name_suffix: ".exe"
             target: x86_64-pc-windows-msvc
     steps:
       - uses: actions/checkout@v3
-      - name: Update rust
+      - name: Update Rust
         run: rustup update
       - name: Add target
         run: rustup target add ${{ matrix.target }}
-      - name: Cache dependencies
-        uses: Swatinem/rust-cache@v2
```

Comment:
Forgot to revert this too

Reply:
3db4e98f711196b631ba0576fdda358f3d84c0c2